libc/kill.c

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <signal.h>
+#include <stddef.h>
+#include <sys/types.h>
+
+/* Panic since there is no support for POSIX signals yet. */
+int kill(pid_t pid, int sig)
+{
```

Comment:
It might be a bad idea to open a issue to track the work for adding  this support in the future.

Reply:
Thanks. I added issue #1870.